packages/core/CHANGELOG.md

```diff
@@ -503,10 +503,10 @@
 
   model.createMachine({
     // `ctx` was of type `any`
-    entry: ctx => {},
+    entry: (ctx) => {},
```

Comment:
We are using a different Prettier version than Changesets and this results in a lot of those reformatting changes. I have a PR open that is supposed to fix this: https://github.com/changesets/changesets/pull/905